### DISCUSSION TITLE
disable fadein when crossfading

### DISCUSSION THREAD

**Discussion by sander816** [AzuraCast/AzuraCast#4627] [Q&A]
is it pssible to disable the fadein when crossfading?
Fade out is ok, but the fadein is a bit odd...

**Comment by Liam024** [comment]
Your Station > Utilities > Edit Liquidsoap Configuration
Edit "fade_in" to 0.00

`radio = crossfade(smart=false,duration=6.00,fade_out=4.00,fade_in=0.00,radio)`

**Comment by sander816** [comment]
where do i edit that? File is uneditable for me?

**Comment by sander816** [comment]
if i edit it with nano, i donot see the changes in the liquidsoap configuration within azuracast

**Reply by Liam024** [reply]
It's working for me:
![image](https://user-images.githubusercontent.com/42640247/134935418-2e8e77df-410d-4db0-9c5c-efa649342a2c.png)
